### PR TITLE
Personalisation

### PR DESCRIPTION
**Pull request by kmalik1978**
Sprint 1 updates

### CODE REVIEW

File: SFA.DFC.TestAutomation/SFA.DFC.MatchYourSkillsToCareer.UITests/Project/Tests/StepDefinitions/DYSACSteps.cs [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+        #region When
+
+        [When(@"I complete the DYSAC (.*) questions and click to see results")]
+        public void WhenICompleteTheDYSACQuestionsAndClickToSeeResults(int p0)
+        {
+            int i = 0;
+            do
+            {
+                dYSACTraitsQuestionsPage
+                    .SelectStronglyAgree()
```
Comment (LJGitHubby) [inline SFA.DFC.TestAutomation/SFA.DFC.MatchYourSkillsToCareer.UITests/Project/Tests/StepDefinitions/DYSACSteps.cs]: Do we have a script which does anything other than select "StonglyAgree" in answer to each question?

Comment (kmalik1978) [reply SFA.DFC.TestAutomation/SFA.DFC.MatchYourSkillsToCareer.UITests/Project/Tests/StepDefinitions/DYSACSteps.cs]: At the moment we don't have this - we are just focusing on getting through the DYSAC questions and back to Match skills. Once we can see the kinds of results we are returning we can tailor this to either a random selector or a pre-defined one.

**Comment by kmalik1978** [comment]
> The Outline Scenario (Use Match Skills To Find Jobs by Entering Occupation Only) seems to describe a test which would stop at check for 'Jobs' after "Then I am on the Results page" step.
> 
> Would (Use Match Skills To View A Job based upon the First skill associated to a chosen Occupation) be more accurate?

Absolutely - the scenarios at the moment are named (more my benefit) on the routes through the application. Once our e2e journey is settled they should be renamed to reflect the actual test as suggested above

File: SFA.DFC.TestAutomation/SFA.DFC.MatchYourSkillsToCareer.UITests/Project/Tests/Pages/MoreJobsPage.cs [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+            _pageHelper = context.Get<PageInteractionHelper>();
+            _objectContext = context.Get<ObjectContext>();
+        }
+
+
+        public MoreJobsPage EnterJob(string job)
+        {
+            _objectContext.Set("searchedTerm", job);
+            _formHelper.EnterText(TextEntryJob, job);
+            System.Threading.Thread.Sleep(2000);
```
Comment (ilambat) [inline SFA.DFC.TestAutomation/SFA.DFC.MatchYourSkillsToCareer.UITests/Project/Tests/Pages/MoreJobsPage.cs]: @kmalik1978 Do we need to use a sleep here? Could you use another method e.g. (WaitForElementToContainText() as an example)?

File: SFA.DFC.TestAutomation/SFA.DFC.MatchYourSkillsToCareer.UITests/Project/Tests/StepDefinitions/DYSACSteps.cs [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+        {
+            dYSACResultsPage = new DYSACResultsPage(_context);
+        }
+
+        [Then(@"I am taken to the DYSAC additional questions page")]
+        public void ThenIAmTakenToTheDYSACAdditionalQuestionsPage()
+        {
+            dYSACAttributesQuestionsPage = new DYSACAttributesQuestionsPage(_context);
+        }
+        #endregion
```
Comment (ilambat) [inline SFA.DFC.TestAutomation/SFA.DFC.MatchYourSkillsToCareer.UITests/Project/Tests/StepDefinitions/DYSACSteps.cs]: Are these placeholders for now? or is there meant to be an assert in these?

Comment (kmalik1978) [reply SFA.DFC.TestAutomation/SFA.DFC.MatchYourSkillsToCareer.UITests/Project/Tests/StepDefinitions/DYSACSteps.cs]: Yes - these are just placeholders at the moment - we haven't integrated the DYSAC service into our tests

File: SFA.DFC.TestAutomation/SFA.DFC.MatchYourSkillsToCareer.UITests/Project/Tests/Pages/MoreJobsPage.cs (hunk 2 above)
Comment (kmalik1978) [reply SFA.DFC.TestAutomation/SFA.DFC.MatchYourSkillsToCareer.UITests/Project/Tests/Pages/MoreJobsPage.cs]: This is just a temporary fudge while the drop downs functionality is ironed out/perfected - at the moment we have some issues with the performance of the drop and i have the sleep just to get us past this page - using the sleep allows me to increase the wait time. But yes, the method suggested above is the way forward.

**Pull request merged**